### ISSUE TITLE
Invalidate cache for a given function

### ISSUE THREAD

**Issue by andrewtmendoza** [peterbe/django-cache-memoize#56]
Hello, is there a way to invalidate all cached values for a function versus invalidating the cached value for a given set of arguments?

**Comment by silentjay** [comment]
No, it mentions this in the readme. However you can clear all cached values on a per function basis with django-memoize
